media-video/ffmpeg/ffmpeg7-7.1.1.recipe

```diff
@@ -26,13 +26,13 @@ ffmpegLibs="\
 	libswscale \
 	"
 # Do not sort alphabetically
-libavutilVersion="59.39.100"
-libavcodecVersion="61.19.100"
-libavformatVersion="61.7.100"
-libavdeviceVersion="61.3.100"
-libavfilterVersion="10.4.100"
-libswscaleVersion="8.3.100"
-libswresampleVersion="5.3.100"
+libavutilVersion="59"
+libavcodecVersion="61"
+libavformatVersion="61"
+libavdeviceVersion="61"
+libavfilterVersion="10"
+libswscaleVersion="8"
+libswresampleVersion="5"
```

Comment:
Did you actually test this? The full version numbers are needed below, e.g. for the debugInfo package. I don't think ffmpeg changed their build system to not generate full soversions any more.